bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/config/CallOptionsConfig.java

```diff
@@ -149,16 +185,34 @@ public Builder setReadRowsRpcTimeoutMs(int readStreamRpcTimeoutMs) {
       return this;
     }
 
+    /**
+     * The amount of time in milliseconds to wait before issuing a client side timeout for readRows
+     * streaming RPCs.
+     *
+     * @param readRowsRpcAttemptTimeoutMs timeout value in milliseconds.
+     * @return a {@link Builder} object, for chaining
```

Comment:
Similar to above, explain that this used for scans and how it differs from the existing non-attempt setting

Reply:
Done